test/jdk/sun/security/pkcs11/Cipher/TestCipherMode.java

```diff
@@ -27,9 +27,29 @@
  * @summary
  * @library /test/lib ..
  * @modules jdk.crypto.cryptoki
- * @run main/othervm TestCipherMode
+ * @run main/othervm TestCipherMode AES/ECB/PKCS5Padding
  */
 
+/*
+ * @test
```

Comment:
```suggestion
 * @test id=AES-GCM
```
Consider adding meaningful IDs to the tests.